quarkus/tests/integration/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java

```diff
@@ -456,7 +461,11 @@ private void setProperty(String key, String value, File confFile) {
         try (
             FileOutputStream out = new FileOutputStream(confFile)
         ) {
-            properties.put(key, value);
+            if (value != null) {
+                properties.put(key, value);
+            } else {
+                properties.remove(key);
```

Comment:
this is unexpected as the method is called `setProperty`, is it needed?

Reply:
Yes it is needed to allow removing properties from the config if needed. Do you want to change the method name? `setOrRemoveProperty`? Isn't enough to accept that a `null` value invalidates a key?

Reply:
I've changed to use `removeProperty`.